app/models/host/base.rb

```diff
@@ -134,6 +134,8 @@ def import_facts(facts)
       # we are not importing facts for hosts in build state (e.g. waiting for a re-installation)
       raise ::Foreman::Exception.new('Host is pending for Build') if build?
 
+      facts[:domain].downcase!
```

Comment:
`undefined method `downcase!' for nil:NilClass` when discovering a host. This breaks Discovery plugin. I alredy finished testing 5.0 with 1.11 so I would rather not see it in the release.